solidjs-tailwind/src/components/UserProfile/UserProfile.jsx

```diff
@@ -0,0 +1,101 @@
+import {
+  users,
+  star,
+  buildingOffice,
+  mapPin,
+  link,
+} from 'solid-heroicons/outline';
+import { Icon } from 'solid-heroicons';
+import { TwitterIcon } from '../Icons';
+import OrgList from './OrgList';
+
+const UserProfile = (userProfileProps) => {
+  return (
+    <div>
+      <img
```

Comment:
this image doesn't seem responsive. Please confirm

Reply:
The image is responsive